cli/tests/test_split_command.rs

```diff
@@ -1125,49 +1123,30 @@ fn test_split_with_message() {
     ]);
     insta::assert_snapshot!(output, @r"
     ------- stderr -------
-    First part: qpvuntsm 6ffbff48 fix in file1
-    Second part: royxmykx 8cfdeaab my feature
-    Working copy  (@) now at: royxmykx 8cfdeaab my feature
-    Parent commit (@-)      : qpvuntsm 6ffbff48 fix in file1
+    First part: royxmykx 6ffbff48 fix in file1
+    Second part: qpvuntsm 8cfdeaab my feature
+    Working copy  (@) now at: qpvuntsm 8cfdeaab my feature
+    Parent commit (@-)      : royxmykx 6ffbff48 fix in file1
     [EOF]
     ");
 
     insta::assert_snapshot!(get_log_output(&work_dir), @r"
-    @  royxmykxtrkr false my feature
-    ○  qpvuntsmwlqt false fix in file1
+    @  qpvuntsmwlqt false my feature
+    ○  royxmykxtrkr false fix in file1
     │
     │  CC: test.user@example.com
     ◆  zzzzzzzzzzzz true
     [EOF]
     ");
 }
 
-enum BookmarkBehavior {
-    Default,
-    MoveBookmarkToChild,
-    LeaveBookmarkWithTarget,
-}
-
-// TODO: https://github.com/jj-vcs/jj/issues/3419 - Delete params when the config is removed.
-#[test_case(BookmarkBehavior::Default; "default_behavior")]
-#[test_case(BookmarkBehavior::MoveBookmarkToChild; "move_bookmark_to_child")]
-#[test_case(BookmarkBehavior::LeaveBookmarkWithTarget; "leave_bookmark_with_target")]
-fn test_split_with_bookmarks(bookmark_behavior: BookmarkBehavior) {
+#[test]
+fn test_split_with_bookmarks() {
     let mut test_env = TestEnvironment::default();
     let edit_script = test_env.set_up_fake_editor();
     test_env.run_jj_in(".", ["git", "init", "main"]).success();
     let main_dir = test_env.work_dir("main");
 
```

Comment:
Can you restore this test? I don't think this addresses the discussion of choosing the correct default for the issue.

Reply:
> Do you mean restoring the two possible bookmark behaviors?

Yes, since it deletes the test required for a currently hidden but supported flag and leaves it untested. 

> The split-out commit may be moved anywhere; I wouldn't expect the bookmark to be attached to that commit. If I have a commit tree like:

yes, that's also the thing I want but I would rather have a separate test for now until the maintainers come to a conclusion for #3419 (I at least posted my final opinion on it after seeing this PR).